jupyterlite_sphinx/jupyterlite_sphinx.py

```diff
@@ -572,6 +572,10 @@ def jupyterlite_build(app: Sphinx, error):
 
         jupyterlite_dir = str(app.env.config.jupyterlite_dir)
 
+        jupyterlite_build_command_options: Dict[str, Any] = (
+            app.env.config.jupyterlite_build_command_options
+        )
```

Comment:
Should we keep this as `[str, str]` since we're not pre-processing any of the values from the dict?
```suggestion
        jupyterlite_build_command_options: Dict[str, str] = (
            app.env.config.jupyterlite_build_command_options
        )
```